staging/src/k8s.io/apiextensions-apiserver/pkg/apiserver/conversion/webhook_converter.go

```diff
@@ -170,14 +172,26 @@ func (c *webhookConverter) Convert(in runtime.Object, toGV schema.GroupVersion)
 	}
 
 	request := createConversionReview(in, toGV.String())
-	if len(request.Request.Objects) == 0 {
+	objCount := len(request.Request.Objects)
+	if objCount == 0 {
 		if !isList {
 			return in, nil
 		}
 		out := listObj.DeepCopy()
 		out.SetAPIVersion(toGV.String())
 		return out, nil
 	}
+
+	trace := utiltrace.New("Call conversion webhook",
+		utiltrace.Field{"custom-resource-definition", c.name},
+		utiltrace.Field{"desired-api-version", request.Request.DesiredAPIVersion},
+		utiltrace.Field{"object-count", objCount},
+		utiltrace.Field{"UID", request.Request.UID})
+	// Only log conversion webhook traces that exceed a 8ms per object limit plus a 50ms request overhead allowance.
+	// The per object limit uses the SLO for conversion webhooks (~4ms per object) plus time to serialize/deserialize
+	// the conversion request on the apiserver side (~4ms per object).
```

Comment:
4ms doesn't seem to match constant 8 in the line below.

Reply:
4+4=8